src/main.h

```diff
@@ -708,6 +722,10 @@ class CBlock
         if (nBestHeight == 0) {
             return CURRENT_BLOCK_VERSION_1;
         }
+        if(nBestHeight >= GetForkHeightTwo()-5)
```

Comment:
-5 is not necessary, all the other changes are at the same height than fork two

Reply:
Thanks

Reply:
oh, wait, I thought you might have changed it, didn't realize these were just comments on past changes xD